### PR TITLE
Distributor ha

### PR DESCRIPTION
**Pull request by eldada**
**What this PR does / why we need it**:
This PR implements true HA for the distribution service.

**Special notes for your reviewer**:
This is a new major version. Not compatible with older chart.

### CODE REVIEW

File: stable/distribution/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # JFrog Distribution Chart Changelog
 All notable changes to this project chart be documented in this file.
 
+## [1.0.0] - Sep 17, 2018
+* True HA with distributor and Redis in their own StatefulSets and headless services
+* Redis StatefulSet now part of the main templates
+* New Redis version: 4.0.11
```
Comment (jainishshah17) [inline stable/distribution/CHANGELOG.md]: @eldada Can you add a note for the backward compatibility/upgrade in README.md

File: stable/distribution/ci/test-values.yaml [hunk 2]
```diff
@@ -1,11 +1,34 @@
...
+#  persistence:
+#    enabled: false
+#    existingClaim:
+  rbac:
+    create: false
+  serviceAccount:
+    create: false
 
 distributor:
   token: "eyJ2ZXIiOiIyIiwidHlwIjoiSldUIiwiYWxnIjoiUlMyNTYiLCJraWQiOiJ6WWFWQjVFRlpkOXlpbWtCNkZTbDAtSWFhMHRPWldLX293c1BTVTNTZmdNIn0.eyJzdWIiOiJJTlRFUk5BTCIsInNjcCI6ImFwcGxpZWQtcGVybWlzc2lvbnNcL3N5c3RlbSIsImF1ZCI6ImpmZHNAMDFjN3pxd2FnOWNmNTAwd2FnOXZhcTF3cjkiLCJpc3MiOiJqZmRzQDAxYzd6cXdhZzljZjUwMHdhZzl2YXExd3I5IiwiZXhwIjoyNDY3NjI1MDczLCJpYXQiOjE1MjE1NDUwNzMsImp0aSI6IjE0ZjJjN2IxLTMzYzQtNDA0Yy04YTllLTJmZjU2OGFjODRlOSJ9.ACfT3pfu4wTTt1wOkyqwYCp9UUVWNwMLl3ZyuO-igDR9N_G-G3oMkuCFekAzgOrTcbGjrXTDMNwoRyYhHUs4wtYxM9yAA-u72EhVz10stL9B0d2JUOamc8xJaLvQDhL38TYiTjdgNVsy726qgxzX2xnx1M8VSp9Qnw7r2ZGBQ6LjrESIUZb8IXy-93oxYqDSkWvjOJZiNQXhJBgOzhzq3r-buaGgxOMoC7DduCctCT0FIxd1jrsZPUnpH3YGE374j7YGcGBdy7eQj-skM3YircJVgk6XFYJBKZBJ8pX3NLZznirG2m9QbhLCRhH1T6B4g-HZvy251wz3QBUdmzO6Cg"
```
Comment (jainishshah17) [inline stable/distribution/ci/test-values.yaml]: @eldada  We will need to add the guide in README for "How to update distributor token"

File: stable/distribution/CHANGELOG.md (hunk 1 above)
Comment (rimusz) [reply stable/distribution/CHANGELOG.md]: +1

Comment (eldada) [reply stable/distribution/CHANGELOG.md]: See new changes to README along with other tunings.

File: stable/distribution/ci/test-values.yaml (hunk 2 above)
Comment (eldada) [reply stable/distribution/ci/test-values.yaml]: We need to have this documented properly in product's Wiki first.

**Pull request merged**